src/form/index.es

```diff
@@ -219,7 +269,8 @@ Form.propTypes = {
   onInvalidSubmit: PropTypes.func,
   onSubmit: PropTypes.func,
 
-  locked: PropTypes.bool,
+  locked: PropTypes.bool.isRequired,
+  defaultLocked: PropTypes.bool.isRequired,
```

Comment:
`lockedByDefault` sounds nicer imo

Reply:
we have a common pattern of prefixing default value for a prop with `default`. I see how yours sounds clearer but for consistency I think it's best to use `defaultSmth`